thirdparties/protobuf/protobuf.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/protobuf.bzl b/protobuf.bzl
+index 4226a1424..36c325aa9 100644
```

Comment:
What does this patch do?

Reply:
> What does this patch do?

For compatibility with higher versions of bazel